xarray/tests/test_dataarray.py

```diff
@@ -3305,6 +3305,45 @@ def test_rank(self):
         y = DataArray([0.75, 0.25, np.nan, 0.5, 1.0], dims=('z',))
         assert_equal(y.rank('z', pct=True), y)
 
+    def test_corr(self):
+        # self: Load demo data and trim it's size
+        ds = xr.tutorial.load_dataset('air_temperature')
```

Comment:
Loading the tutorial datasets requires network access, which we try to avoid for tests. Can you write this test using synthetic data instead?

Reply:
Yes, will do. The tutorial code can be moved into an 'example' in documentation/ user guide later.